docker-compose/setup-script/noco.sh

```diff
@@ -643,17 +611,58 @@ cat <<EOF >> docker-compose.yml
       - /bin/sh
       - -c
       - redis-server --requirepass "\$\${REDIS_PASSWORD}"
-    volumes:
-      - redis:/data
     healthcheck:
       test: [ "CMD", "redis-cli", "-a", "\$\${REDIS_PASSWORD}", "--raw", "incr", "ping" ]
+    volumes:
+      - ./redis:/data
     networks:
       - nocodb-network
 EOF
+    fi
+
+# IF Minio is enabled add the following lines to the compose file
+    if [ "${CONFIG_MINIO_ENABLED}" = "Y" ]; then
+        cat >> "$compose_file" <<EOF
+  minio:
+    image: minio/minio:latest
+    restart: unless-stopped
+    env_file: docker.env
+    entrypoint: /bin/sh
+    volumes:
+      - ./minio:/export
+    command: -c 'mkdir -p /export/nocodb && /usr/bin/minio server /export'
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.services.minio.loadbalancer.server.port=9000"
+      - "traefik.http.routers.minio.rule=Host(\`${CONFIG_MINIO_DOMAIN_NAME}\`)"
+EOF
+# If minio SSL is enabled, set the entry point to websecure
 fi
+    if [ "${CONFIG_SSL_ENABLED}" = "Y" ]; then
```

Comment:
I guess we have to check `CONFIG_MINIO_SSL_ENABLED` here

Reply:
updated